static/js/components/AnnotationsTable.jsx

```diff
@@ -224,15 +271,51 @@ const AnnotationsTable = ({ annotations, spectrumAnnotations = [] }) => {
     jumpToPage: false,
     pagination: true,
     tableBodyMaxHeight: "20rem",
+    customToolbar: () => (
+      <>
```

Comment:
I might be wrong but not sure these `<> </>` are needed here right?

Reply:
good.